packages/webdriver/src/monad.js

```diff
@@ -68,30 +77,60 @@ export default function WebDriver (options, modifier, propertiesObject = {}) {
                 ? commandWrapper(name, func)
                 : func
             if (attachToElement) {
+                /**
+                 * add command to every multiremote instance
+                 */
                 if (instances) {
                     Object.values(instances).forEach(instance => {
                         instance.__propertiesObject__[name] = {
                             value: customCommand
                         }
                     })
                 }
+
                 this.__propertiesObject__[name] = { value: customCommand }
             } else {
                 unit.lift(name, customCommand, proto)
             }
         }
 
+        client.overwriteCommand = function (name, func, attachToElement = false, proto, instances) {
```

Comment:
Would be possible to overwrite e.g. `click()`? And use it like `element.click()`. Or it is just for `BrowserObject`?